front/src/reducers/osrdconf/osrdConfCommon/__tests__/utils.ts

```diff
@@ -600,6 +602,224 @@ const testCommonConfReducers = (slice: OperationalStudiesConfSlice | StdcmConfSl
     const state = defaultStore.getState()[slice.name];
     expect(state.trainScheduleIDsToModify).toStrictEqual(newTrainScheduleIDsToModify);
   });
+
+  it('should handle updateOriginV2', () => {
+    const newOrigin = testDataBuilder.buildPathSteps()[0];
+    defaultStore.dispatch(slice.actions.updateOriginV2(newOrigin));
+    const state = defaultStore.getState()[slice.name];
+    expect(state.pathSteps[0]).toEqual(newOrigin);
+  });
+
+  it('should handle updateDestinationV2', () => {
+    const newDestination = testDataBuilder.buildPathSteps()[2];
+    defaultStore.dispatch(slice.actions.updateDestinationV2(newDestination));
+    const state = defaultStore.getState()[slice.name];
+    expect(last(state.pathSteps)).toEqual(newDestination);
+  });
+
+  it('should handle deleteItineraryV2', () => {
+    const pathSteps = testDataBuilder.buildPathSteps();
+    const store = createStore(slice, {
+      pathSteps,
+    });
+    store.dispatch(slice.actions.deleteItineraryV2());
+    const state = store.getState()[slice.name];
+    expect(state.pathSteps).toEqual([null, null]);
+  });
+
+  it('should handle clearViasV2', () => {
+    const pathSteps = testDataBuilder.buildPathSteps();
+    const store = createStore(slice, {
+      pathSteps,
+    });
+    store.dispatch(slice.actions.clearViasV2());
+    const state = store.getState()[slice.name];
+    expect(state.pathSteps).toEqual([pathSteps[0], last(pathSteps)]);
+  });
+
+  it('should handle deleteViaV2', () => {
+    const pathSteps = testDataBuilder.buildPathSteps();
+    const store = createStore(slice, {
+      pathSteps,
+    });
+    store.dispatch(slice.actions.deleteViaV2(0));
+    const state = store.getState()[slice.name];
+    expect(state.pathSteps).toEqual(removeElementAtIndex(pathSteps, 1));
+  });
+
+  describe('should handle addViaV2', () => {
+    const pathStepsData = testDataBuilder.buildPathSteps();
+    const [brest, rennes, lemans, paris, strasbourg] = pathStepsData;
+
+    const rennesVia = testDataBuilder.buildPointOnMap({
+      id: 'rennes',
+      coordinates: [48.10326700633057, -1.6719908615098822],
+      location: {
+        track_section: '697841c6-6667-11e3-81ff-01f464e0362d',
+        offset: 233.404,
+      },
+    });
+    const lemansVia = testDataBuilder.buildPointOnMap({
+      id: 'lemans',
+      coordinates: [47.99542250806296, 0.1918181738752042],
+      location: {
+        track_section: '60ca8dda-6667-11e3-81ff-01f464e0362d',
+        offset: 426.443,
+      },
+    });
+    const parisVia = testDataBuilder.buildPointOnMap({
+      id: 'paris',
+      coordinates: [48.904852473668086, 2.4369545094357736],
+      location: {
+        track_section: '63c905ee-6667-11e3-81ff-01f464e0362d',
+        offset: 719.258,
+      },
+    });
```

Comment:
Can you move the 3 const in the testDataBuilder ? :)

Reply:
buildPointOnMap is already a dataBuilder method so I thought it was the good way to use it in a test.

Do you prefer if I create a buildPointOnMaps returning these 3 objects or you just want these 3 consts in a different file ?

Reply:
As you wish, I don't really mind :) Maybe in dataBuilder, since it already returns static objects (that don't change) ?